contracts/specifications/arbitrator.md

```
@@ -592,55 +582,60 @@ function draw(uint256 _disputeID, uint256 _iterations) external
-  - Dispute kit jumps when parent court compatibility requires
-  - Creation of new rounds
-  - Fee payments and stake calculations
+**New features**:
+- Improved court/DK jump logic via `_getCompatibleNextRoundSettings()`
+- Better compatibility validation
+
+## Internal Mechanics (implementation detail)
+
+The following are internal functions and implementation details that are subject to change. Do not depend on these externally:
+
+#### _getCompatibleNextRoundSettings
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)